spec/support/share_examples/wallets_controller_spec.rb

```diff
@@ -3,16 +3,30 @@
     before do
       get action
     end
-    it "redirect login page"do
-      expect(response).to redirect_to login_url
+    it "returns	a	302	response"do
```

Comment:
spaces chỗ này như nào đấy em, sửa lại hết trong file này đi nhé
```suggestion
    it "returns a 302 response"do
```

Reply:
Dạ do em dùng máy cá nhân nên nó bị lỗi spaces, em cũng không biết sửa sao khi mà chuyển qua máy cá nhân. Lâu lâu nó lại bị vậy chắc máy laptop sắp đi rồi ạ.